### PR TITLE
refact(upgrade): added newline to comments to make the yaml readable

### PR DESCRIPTION
**Pull request by shubham14bajpai**
Signed-off-by: shubham <shubham.bajpai@mayadata.io>

**What this PR does / why we need it:**
 - This PR adds new line to comments in the job yamls to make them more readable.
 - It also corrects pv name comment in jiva upgrade yaml.

### CODE REVIEW

File: k8s/upgrades/1.0.0-1.1.0/example-cstor-spc-upgrade-job.yaml [hunk 1]
```diff
@@ -6,35 +6,47 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
+  #
   #VERIFY that you have provided a unique name for this upgrade job.
   #The name can be any valid K8s string for name. This example uses
   #the following convention: cstor-spc-<flattened-from-to-versions>-<spc-name>
+  #
   name: cstor-spc-100110-cstor-sparse-pool
+  #
```
Comment (AmitKumarDas) [inline k8s/upgrades/1.0.0-1.1.0/example-cstor-spc-upgrade-job.yaml]: can this be a newline only

File: k8s/upgrades/1.0.0-1.1.0/example-cstor-spc-upgrade-job.yaml [hunk 2]
```diff
@@ -6,35 +6,47 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
+  #
   #VERIFY that you have provided a unique name for this upgrade job.
```
Comment (AmitKumarDas) [inline k8s/upgrades/1.0.0-1.1.0/example-cstor-spc-upgrade-job.yaml]: Can we have a space between # and comment

File: k8s/upgrades/1.0.0-1.1.0/example-cstor-spc-upgrade-job.yaml [hunk 3]
```diff
@@ -6,35 +6,47 @@
...
   #The name can be any valid K8s string for name. This example uses
   #the following convention: cstor-spc-<flattened-from-to-versions>-<spc-name>
+  #
```
Comment (AmitKumarDas) [inline k8s/upgrades/1.0.0-1.1.0/example-cstor-spc-upgrade-job.yaml]: Did we try below:
```yaml
# Some comments that explain the field below
field: "Hello"

# Comments that explain newfield
newfield: "Did you try"
```

File: k8s/upgrades/1.0.0-1.1.0/example-cstor-spc-upgrade-job.yaml (hunk 2 above)
Comment (shubham14bajpai) [reply k8s/upgrades/1.0.0-1.1.0/example-cstor-spc-upgrade-job.yaml]: Done

File: k8s/upgrades/1.0.0-1.1.0/example-cstor-spc-upgrade-job.yaml (hunk 3 above)
Comment (shubham14bajpai) [reply k8s/upgrades/1.0.0-1.1.0/example-cstor-spc-upgrade-job.yaml]: Done

**Pull request merged**